test/test_van/test_people.py

```diff
@@ -60,6 +60,53 @@ def test_valid_search(self):
         self.van._valid_search('Barack', 'Obama', None, 2024291000, None, None,
                                None, None, None)
 
+        # Successful with match_map FN/LN/Email
+        match_map_1 = {'firstName': 'Barack', 'lastName': 'Obama', 'emails': [{'email': 'barack@email.com'}]}
+
+        self.van._valid_search(None, None, None, None, None, None,
+                               None, None, match_map_1)
+
+        # Successful with match_map FN/LN/Phone
+        match_map_2 = {'firstName': 'Barack', 'lastName': 'Obama', 'phones': [{'phoneNumber': 2024291000}]}
+
+        self.van._valid_search(None, None, None, None, None, None,
+                               None, None, match_map_2)
+
+        # Successful with match_map FN/LN/DOB/ZIP
+        match_map_3 = {'firstName': 'Barack', 'lastName': 'Obama', 'addresses': [{'zipOrPostalCode': 20009}], 'dateOfBirth': '2000-01-01'}
+
+        self.van._valid_search(None, None, None, None, None, None,
+                               None, None, match_map_3)
+
+        # Successful with match_map FN/LN/STREETNAME/STREETNUMBER/ZIP
+        match_map_4 = {
+            'firstName': 'Barack',
+            'lastName': 'Obama',
+            'addresses':
+            [{
+                'zipOrPostalCode': 20009,
+                'addressLine1': 'glenwood drive'
```

Comment:
This will not be helpful as part of these testing functions until we replace all instances of `street_number` and `street_name` arguments in the people method with `addressLine1`. There is currently nothing in our own validation process that recognizes street name or number from `addressLine1`.